mobile/src/bundles/common/components/selector/selector.tsx

```diff
@@ -44,8 +43,7 @@ const Selector: React.FC<Properties> = ({ label, options }) => {
         : IconName.ARROW_DROP_DOWN;
 
     return (
-        <View>
-            <Text category={TextCategory.LABEL}>{label}</Text>
+        <View style={{ position: 'relative' }}>
```

Comment:
remove inline styles (move to styles.ts)

Reply:
done